templates/gitlab/myrror-cli.gitlab-ci.yml

```diff
@@ -4,8 +4,34 @@ myrror-scan:
   before_script:
     - MYRROR_REPOSITORY=$CI_PROJECT_PATH
     - MYRROR_ROOT_NAMESPACE=$CI_PROJECT_ROOT_NAMESPACE
+    - apt-get -qq update
+    - apt-get install -y jq
+    - |
+      if [ "$CI_PIPELINE_SOURCE" == "merge_request_event" ]; then
+        MR_DETAILS=$(curl --header "PRIVATE-TOKEN: $ACCESS_TOKEN" "https://gitlab.com/api/v4/projects/$CI_PROJECT_ID/merge_requests/$CI_MERGE_REQUEST_IID")
```

Comment:
personal access token required to call Gitlab API in CI. There is [CI_JOB_TOKEN](https://docs.gitlab.com/ee/ci/jobs/ci_job_token.html) in CI predefined env variables, but that don't have access to [Merge Requests API](https://docs.gitlab.com/ee/api/merge_requests.html#get-single-mr)